### ISSUE TITLE
BUG: interpreter not working

### ISSUE THREAD

**Issue by snam24** [obsidianmd/obsidian-clipper#241]
**Version (please complete the following information):**

- OS: [e.g. Windows] Mac, Android
- Browser [e.g. Chrome, Firefox] Edge, Firefox
- Web Clipper version: [e.g. 0.9.5] 0.10.2, 0.10.4
- Obsidian version: [e.g. 1.7.5] 1.7.7

**Describe the bug**

The interpreter doesn't work when I click the (red or purple) "interpret" button on the clipper.
![Screenshot_2024-12-08-21_08_17_D84dWG](https://github.com/user-attachments/assets/a376af54-5435-47ea-b858-f548ae9fd824)

Screen recording
https://github.com/user-attachments/assets/fbf8a813-43e3-4de2-9f44-d94036d80d3d

**Expected behavior**

It used to work several days ago when I first installed the plugin. I had used pre-set models (GPT 4o or GPT 4o mini) and provided an OpenAI API. After an update (?), I had to add a provider, so I did. I added a model too as below. However, the interpreter doesn't work. I tried `gpt-4o-mini-2024-07-18` model ID too, which doesn't work either.

![Screenshot_2024-12-08-21_16_33_ZE3YuZ](https://github.com/user-attachments/assets/0b3db46d-bc6a-4097-a923-f0415cba0a11)

**URLs where the bug occurs**

Every webpage.

**To reproduce**

1. Go to '...' (any website)
2. Click on 'obsidian clipper'
3. Click 'interpret'
4. Nothing happens

If you are using a custom template (i.e. not the Default template). Go to Web Clipper settings and click **More** → **Copy as JSON**. Paste the JSON code below.

```json
{
  "schemaVersion": "0.1.0",
  "name": "web_clipper_with_ai_is_chefs_kiss",
  "behavior": "create",
  "noteContentFormat": "## Summary\n{{\"A summary of the page. Cover everything important, be verbose if needed. \"}}\n*(summary generated by GPT 4o)*\n\n# Content\n{{content}}",
  "properties": [
    {
      "name": "title",
      "value": "{{title}}",
      "type": "text"
    },
    {
      "name": "link",
      "value": "{{url}}",
      "type": "text"
    },
    {
      "name": "authors",
      "value": "{{\\\"Authors of this page\\\"|split:\\\", \\\"|wikilink|join}}",
      "type": "multitext"
    },
    {
      "name": "published",
      "value": "{{published}}",
      "type": "date"
    },
    {
      "name": "created",
      "value": "{{date}}",
      "type": "date"
    },
    {
      "name": "description",
      "value": "{{\\\"A brief summary of this page\\\"}}",
      "type": "text"
    },
    {
      "name": "tags",
      "value": "clippings,{{\\\"1 or 2 words in lowercase describing the type of this page. a type can be multiple words if hyphenated\\\"}}",
      "type": "multitext"
    },
    {
      "name": "sources",
      "value": "{{\\\"Source of this page: the group, organization or individual. If multiple, use as a comma-separated list\\\"|split:\\\", \\\"|wikilink|join}}",
      "type": "multitext"
    },
    {
      "name": "topics",
      "value": "{{\\\"Generate a list of less than 5 tags that cover the main topics on this page. Words must begin with a capital letter..\\\"|split:\\\", \\\"|wikilink|join}}",
      "type": "text"
    }
  ],
  "triggers": [],
  "noteNameFormat": "{{\"provide a title for this page in 10 words or less\"}} ",
  "path": "Clippings"
}
```

**Comment by kepano** [comment]
You need to select a model in the list by clicking the area on the left.

The bug is that it's possible to disable a model and then the select element in the popup appears empty. This will be fixed in the next update.

**Comment by kepano** [comment]
Fixed in 0.10.5
